api/src/wfl/api/handlers.clj

```diff
@@ -81,7 +81,8 @@
             (workloads/load-workloads tx)))))))
 
 (defn post-start
-  "Start the workloads with UUIDs in REQUEST."
+  "Start the workload with UUID in REQUEST.
+   Can take either a single UUID or a map with UUID as key."
```

Comment:
Just take one or the other - don't take both!

Reply:
per our discussion [here](https://broadinstitute.slack.com/archives/CFA6Q2QBU/p1603916497140900), will update this.